marshmallow/validate.py

```diff
@@ -12,7 +12,23 @@
 from marshmallow.exceptions import ValidationError
 
 
-class URL(object):
+class Validator(object):
+    """Base abstract class for validators."""
+
+    def __repr__(self):
+        args = self._repr_args()
+        args = '{0}, '.format(args) if args else ''
+
+        return (
+            '<validate.{self.__class__.__name__}({args}error={self.error!r})>'
```

Comment:
I'm going to remove the `validate.` prefix to account for user validators can subclass `Validator`